### ISSUE TITLE
can't build the toolchain.

### ISSUE THREAD

**Issue by Manish-Kala-Jaat** [mohamed-anwar/Aquila#27]
building the toolchain giving errors.

![Screenshot (3701)](https://user-images.githubusercontent.com/49841018/56889133-414fdb00-6a93-11e9-82ee-9f3cf693d168.png)

![Screenshot (3706)_LI](https://user-images.githubusercontent.com/49841018/56890004-d7850080-6a95-11e9-80be-53d752d6d998.jpg)

![Screenshot (3707)](https://user-images.githubusercontent.com/49841018/56890053-f84d5600-6a95-11e9-9858-8cb4baff41f4.png)

**Comment by mohamed-anwar** [comment]
Hello @Manish-Kala-Jaat 
I'm not sure what could be causing the issue, please check if `build-tools/sysroot/bin` exists and show the binaries in it.
I confess that the current way of building the tool-chain is not a very mature one, I'll try to build packages for different systems to ease-out development and testing and update the Wiki with them.

**Comment by Manish-Kala-Jaat** [comment]
yes it exists 

![Screenshot (5419)](https://user-images.githubusercontent.com/49841018/57025647-6724dd80-6c55-11e9-86b9-3e8f1ee7b2b1.png)

**Comment by mohamed-anwar** [comment]
Oh, this is a very old release, could you please clone the master branch and try building from it following the same steps and report if it fails too?

**Comment by Manish-Kala-Jaat** [comment]
I tried  that one too but that one is also not building . It's late night
here , I will share the screenshot by tomorrow.


बुध, 1 मई 2019, 9:24 pm को Mohamed Anwar <notifications@github.com> ने लिखा:

**Comment by Manish-Kala-Jaat** [comment]
yeah man it compiled right now , but results are not as expectations =
I entered the username as root and password as just enter.
when I given the command of lua, nuklear it shows command not found.

![Screenshot (5424)](https://user-images.githubusercontent.com/49841018/57057421-2ca75900-6cc6-11e9-9c84-8a76516559f6.png)

**Comment by mohamed-anwar** [comment]
Hello @Manish-Kala-Jaat 
Yes, sorry for the disappointment. They were distributed as binaries before, they are no longer in the master branch, you can still build them but you'll need to build aquila targeted cross-compiler.

I'll try to add the required patches to build a targeted cross-compiler to the wiki and probably provide instructions to build lua. Nuklear however, was only experimental, and is not by any means a mature windowing systems so I had to remove it until I find a way to do it correctly, unfortunately I have been busy ever since and couldn't really tackle the problem. I'll probably work on it again soon so keep an eye for updates.

For now though, you can try any of the released ISO images and browse the source code for these releases to see how Nuklear was implemented before.
